### PR TITLE
Add status bar item to extension

### PR DESCRIPTION
**Pull request by trevor-scheer**
This PR adds a status bar item to the VSCode extension. It has a loading / loaded state.

Included some named import cleanup as well.

### CODE REVIEW

File: packages/apollo-vscode/src/StatusBar.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    this.statusBarItem.show();
+  }
+
+  public showLoadedState() {
+    if (!window.activeTextEditor) {
+      this.statusBarItem.hide();
+      return;
+    }
+
+    this.statusBarItem.text = "Apollo GraphQL $(rocket)";
```
Comment (jbaxleyiii) [inline packages/apollo-vscode/src/StatusBar.ts]: whoa, is this how you do emoji's with vscode?

Comment (trevor-scheer) [reply packages/apollo-vscode/src/StatusBar.ts]: Yeah! These are the icons vscode makes available:
https://octicons.github.com/

Supports regular emoji too, just paste into code and all set 👍

**Comment by trevor-scheer** [comment]
Shown below: the status bar item with loading and loaded state as VSCode boots up.

statusbar

**Comment by trevor-scheer** [comment]
@jbaxleyiii w/o much time spent looking into this, it seems Chromatic is driven by Storybook and exclusive to React, but we can still get some unit/integration testing around the extension.

Opened an issue for this: https://github.com/apollographql/apollo-cli/issues/609

File: packages/apollo-vscode/src/extension.ts [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+import {
+  window,
+  workspace,
+  ExtensionContext,
+  ViewColumn,
+  WebviewPanel,
+  Uri,
+  ProgressLocation,
+  DecorationOptions
+} from "vscode";
```
Comment (abernix) [inline packages/apollo-vscode/src/extension.ts]: Thank you for making these more explicit, @trevor-scheer!

**Pull request merged**